neuraxle/base.py

```diff
@@ -752,6 +753,30 @@ def get_hyperparams_space(self) -> HyperparameterSpace:
         """
         return self.hyperparams_space
 
+    def apply_method(self, method: Callable, *kargs) -> 'BaseStep':
+        """
+        Apply a method to a step and its children.
+
+        :param method: method to call with self
+        :param kargs: any additional arguments to be passed to the method
+        :return:
+        """
+        method(self, *kargs)
+        return self
+
+    def apply(self, method_name: str, *kargs) -> 'BaseStep':
```

Comment:
also add `**kwargs` for positional named arguments after `*kargs`. Same for the other apply method.